apollo-compiler/src/test/graphql/com/example/enums_with_reserved_words/schema.sdl

```diff
@@ -0,0 +1,8 @@
+type Query {
+  enum: Enum
+}
+
+enum Enum {
+  name
+  ordinal
+}
```

Comment:
What about an integration test instead? In `tests/enums`? That would allow to verify that the generated code compiles but also that the enum value is correctly generated as `name_`

Reply:
Done in 13fd5f010f021b12a76c0d44b53a5ef1a48d1d16 - makes a smaller PR 👍